care/facility/management/commands/clean_patient_disease_status.py

```diff
@@ -0,0 +1,29 @@
+import json
+from django.core.management.base import BaseCommand
+from care.facility.models.patient import PatientRegistration
+
+
+def change_disease_status(prev, new, qs):
+    failed = []
+    for patient in qs:
+        if patient.disease_status == prev:
+            print(patient.disease_status)
+            try:
+                patient.disease_status = new
+                patient.save()
+            except Exception:
+                failed.append({"id": patient.id, "disease_status": patient.disease_status})
+
+            print(patient.disease_status)
+    return failed
+
+
```

Comment:
we can just do patient.filter(disease_status=5).update(disease_status=4) right ?

Reply:
> we can just do patient.filter(disease_status=5).update(disease_status=4) right ?

Yes we can but then we will not know updation of which rows are failed.

Reply:
there will be one single mysql query, what failure are we talking about here ?